website/docs/r/tenant.html.md

```diff
@@ -9,6 +9,11 @@ description: |-
 
 With this resource, you can manage Auth0 tenants, including setting logos and support contact information, setting error pages, and configuring default tenant behaviors.
 
+#### Notes:
+Auth0 does not currently support creating or destryoying tenants through their API. This resource can only manage an existing tenant created through the Auth0 dashboard. 
+
+Auth0 does not currently support adding/removing extensions on tenants through their API. The Auth0 dashboard must be used to add/remove extensions. 
+
```

Comment:
I'm not sure whether the tenant resource is the best place to communicate this message. I would rather leave this out.

Reply:
I wavered on this too. I'm not sure of the best place to put this either. Maybe at the provider level documentation?

I do think it is important information to pass along. Just today I have been working with the Auth0 GitHub deployments, and again thought it would be good to know I cannot add/configure this extension through the auth0 provider.

I leave it up to you as to whether this warrants a note somehwhere, or not.

Thank you so much for your contribution with this provider. I hope to some day be able to make a more meaningful contribution.

Reply:
Thanks for the feedback. I'll give some thought on where it's best to mention it.